pkg/motor/login.go

```diff
@@ -39,6 +39,7 @@ func (mtr *motorNodeImpl) Login(request mt.LoginRequest) (mt.LoginResponse, erro
 	}
 
 	// fetch DID document from chain
+	mtr.callback.OnMotorEvent("Verifying with Blockchain", false)
```

Comment:
References to the value represented by each string should be made through some kind of strongly typed value / constant